packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -14,28 +14,31 @@ export const fontFamilies = {
 } as const;
 
 export const lineHeights = {
-  "0": "8px",
-  "1": "10px",
-  "2": "11px",
-  "3": "12px",
-  "4": "16px",
-  "5": "22px",
+  "0": "16px",
+  "1": "8px",
+  "2": "12px",
+  "3": "10px",
+  "4": "39px",
+  "5": "11px",
   "6": "27px",
-  "7": "39px",
-  "8": "58px",
+  "7": "58px",
+  "8": "22px",
   "9": "360px",
 } as const;
```

Comment:
@taylornowotny 

Can we use "line height", "font size", "font weight", and "letter spacing" tokens in code directly instead of via composed "typography" tokens? Or are these low level properties of text are only for Figma and shouldn't be used in code.

If we want to share them between code and Figma we need to be careful with changes like this.